### PR TITLE
nodePackages.graphql-language-service-cli: init at 3.2.26

### PR DESCRIPTION
**Pull request by Kranzes**
###### Description of changes

Added the npm package `graphql-language-service-cli` and `graphql`.

`graphql` in this case is a library, which is usually recommended against adding to the package set. The reason I decided to package it is, because it is needed in order for other node `graphql` packages like  `graphql-language-service-cli` to work. node2nix does not pick up the `graphql` dependency since it is set as a peer dependency. Therefore I packaged the library as well as the cli tool and wrapped the latter with the former.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by max-privatevoid** [comment]
Result of `nixpkgs-review pr 176463` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>teleprompter</li>
    <li>timedoctor</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>weylus</li>
  </ul>
</details>
<details>
  <summary>65 packages built:</summary>
  <ul>
    <li>antennas</li>
    <li>antora</li>
    <li>balanceofsatoshis</li>
    <li>bitwarden-cli</li>
    <li>breitbandmessung</li>
    <li>castnow</li>
    <li>code-server</li>
    <li>commitlint</li>
    <li>create-cycle-app</li>
    <li>deltachat-desktop</li>
    <li>epgstation</li>
    <li>fast-cli</li>
    <li>flexoptix-app</li>
    <li>gnomeExtensions.pop-shell</li>
    <li>google-clasp</li>
    <li>grafana-image-renderer</li>
    <li>gtop</li>
    <li>hyperspace-cli</li>
    <li>imapnotify</li>
    <li>imgbrd-grabber</li>
    <li>iosevka</li>
    <li>joplin</li>
    <li>lessc</li>
    <li>mastodon-bot</li>
    <li>morgen</li>
    <li>node2nix</li>
    <li>pm2</li>
    <li>postcss-cli</li>
    <li>pulp</li>
    <li>pyright</li>
    <li>redoc-cli</li>
    <li>reveal-md</li>
    <li>shepherd</li>
    <li>shout</li>
    <li>slack</li>
    <li>sloc</li>
    <li>teams</li>
    <li>teck-programmer</li>
    <li>theLoungePlugins.plugins.closepms</li>
    <li>theLoungePlugins.plugins.giphy</li>
    <li>theLoungePlugins.themes.flat-blue</li>
    <li>theLoungePlugins.themes.flat-dark</li>
    <li>thelounge</li>
    <li>triton</li>
    <li>uppy-companion</li>
    <li>vimPlugins.YouCompleteMe</li>
    <li>vimPlugins.coc-explorer</li>
    <li>vimPlugins.coc-html</li>
    <li>vimPlugins.coc-metals</li>
    <li>vimPlugins.coc-pyright</li>
    <li>vimPlugins.coc-snippets</li>
    <li>vimPlugins.coc-stylelint</li>
    <li>vimPlugins.coc-tsserver</li>
    <li>vimPlugins.coc-vetur</li>
    <li>vscode</li>
    <li>vscode-extensions.matklad.rust-analyzer (vscode-extensions.rust-lang.rust-analyzer)</li>
    <li>vscode-extensions.ms-python.vscode-pylance</li>
    <li>vscode-extensions.vadimcn.vscode-lldb</li>
    <li>vscode-fhs</li>
    <li>vscode-with-extensions</li>
    <li>vscodium</li>
    <li>vscodium-fhs</li>
    <li>whalebird</li>
    <li>ycmd</li>
    <li>zx</li>
  </ul>
</details>

**Review by malob — APPROVED** [review]
LGTM.

**Comment by dotlambda** [comment]
Result of `nixpkgs-review pr 176463` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

File: pkgs/development/node-packages/overrides.nix [hunk 1]
```diff
@@ -148,6 +148,14 @@ final: prev: {
     meta = oldAttrs.meta // { broken = since "10"; };
   });
 
+  graphql-language-service-cli = prev.graphql-language-service-cli.override {
+    nativeBuildInputs = [ pkgs.makeWrapper ];
+    postInstall = ''
+      wrapProgram "$out/bin/graphql-lsp" \
+      --prefix NODE_PATH : ${final.graphql}/lib/node_modules
```
Comment (dotlambda) [inline pkgs/development/node-packages/overrides.nix:155]:
```suggestion
        --prefix NODE_PATH : ${final.graphql}/lib/node_modules
```

**Comment by dotlambda** [comment]
@ofborg eval

**Comment by Kranzes** [comment]
Nope, it's fine without it.

**Comment by dotlambda** [comment]
Please rebase on master.

**Review by dotlambda — CHANGES_REQUESTED** [review]
Run `generate.sh` again.

**Comment by Kranzes** [comment]
Will do, this is gonna take a couple of hours.

File: pkgs/development/node-packages/node-packages.nix [hunk 2]
```diff
@@ -64226,7 +64262,7 @@ let
       version = "3.0.2";
       src = fetchurl {
         url = "https://registry.npmjs.org/structured-source/-/structured-source-3.0.2.tgz";
-        sha1 = "dd802425e0f53dc4a6e7aca3752901a1ccda7af5";
+        sha512 = "Ap7JHfKgmH40SUjumqyKTHYHNZ8GvGQskP34ks0ElHCDEig+bYGpmXVksxPSrgcY9rkJqhVMzfeg5GIpZelfpQ==";
```
Comment (dotlambda) [inline pkgs/development/node-packages/node-packages.nix:64265]: I wonder why so many hashes were changed from `sha1` to `sha512`.

**Pull request merged**